src/d3kit-gridmap.js

```diff
@@ -6,6 +6,7 @@ var DEFAULT_OPTIONS = {
   initialWidth: 500,
   initialHeight: 340,
   keyFn: undefined,
+  durationFn: function() {return 250;},
```

Comment:
yes a name change is in order.  i made it a function, as in my case the vis should transition slowly during normal operation, and quickly during the race animation.  if you look [here](http://interactive-staging--prod--batcave.service.smf1.twitter.biz/candidateRace16/index.html) you can see i have the "fast" speed a little too slow (100ms) and the colors get muddy.  at 250ms you can't really tell what's happening at all.